### ISSUE TITLE
SwnMf6 obj has no attribute 'grid_cells'

### ISSUE THREAD

**Issue by wkitlasten** [mwtoews/surface-water-network#21]
Generating my SwnMf6 object and pickling it like this:

```
ngwf = swn.SwnMf6.from_swn_flopy(n, gwf,reach_include_fraction=0)
ngwf.to_pickle('test_pickle.pkl')
```

Then reading it back in like this:

`ngwf = swn.SwnMf6.from_pickle('test_pickle.pkl',gwf)`

But some of the object attributes are note being set. I can manually set `ngwf._swn=n`, but the grid_cells seems a bit more difficult. Results in this error:

AttributeError: 'SwnMf6' object has no attribute 'grid_cells'
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-40-730f2f542035> in <module>
----> 1 ngwf.plot()

d:\modelling\surface-water-network\swn\modflow\_base.py in plot(self, column, cmap, colorbar)
    996             column=column, label="reaches", legend=colorbar, ax=ax, cmap=cmap)
    997 
--> 998         self.grid_cells.plot(ax=ax, color="whitesmoke", edgecolor="gainsboro")
    999 
   1000         def getpt(g, idx):

AttributeError: 'SwnMf6' object has no attribute 'grid_cells'

**Comment by mwtoews** [comment]
Thanks for the report. I've been thinking about this one too, and it seems that `swn.SwnMf6.from_pickle` is not complete. There are two options:
1. Change the signature from `swn.SwnMf6.from_pickle(path, model)` to `(path, swn, model)`
2. Or add the `swn` data to the pickled object, which will increase the convenience, but also the file size.
As for `grid_cells`, this is probably an oversight that should be addressed by including the structure.

**Comment by mwtoews** [comment]
@wkitlasten this should be mostly fixed with #26. One adjustment is that you may need to do is:
```python
ngwf = swn.SwnMf6.from_pickle('test_pickle.pkl', model=gwf)
```
or pass it the swn object as the first argument. This can be kept/loaded in a separate pickle.

Also, `ngwf.swn = n` will be the preferred way to set this property, where needed.
